pilot/utils/arguments.py

```diff
@@ -36,12 +39,10 @@ def get_arguments():
         arguments['app_id'] = str(uuid.uuid4())
 
     if 'user_id' not in arguments:
-        arguments['user_id'] = str(uuid.uuid4())
+        arguments['user_id'] = getpass.getuser()
```

Comment:
since https://github.com/Pythagora-io/gpt-pilot/pull/57 is merged we dont need this, right?

Reply:
yeah, it must have leaked in from the other branch